collection_requirements.rst

```diff
@@ -167,6 +167,8 @@ We should avoid FQCN / repository names:
 * which are unnecessary long: try to make it compact but clear
 * contain the same words / collocations in ``NAMESPACE`` and ``COLLECTION`` parts, for example ``my_system.my_system``
 
+If your collection is planned to be certified on Automation Hub, please consult with Red Hat Partner Engineering to ensure collection naming compatibility between the community collection on **Galaxy**.
```

Comment:
That's saying something different from what we decided on the community meeting. I think this should explicitly say that the names should be identical if possible.
```suggestion
If your collection is planned to be certified on Automation Hub, please consult with Red Hat Partner Engineering to ensure collection naming compatibility between the community collection on **Galaxy**. It is desirable that the collection uses the same name on both platforms.
```

Reply:
I must agree with Felix here. "Naming compatibility" is far weaker than "use the same name or end users will have to rewrite playbooks when migrating between Galaxy and AH content".

Reply:
In most cases this policy is not something that the Community Team (or Ansible) has any control over. There are branding implications and best practices that need to be maintained across upstream and downstream. I would leave it to just having the submitter to talk to the Partner Engineering team regarding naming, as the only folks affected are probably Red Hat Collections.